.github/workflows/nightly.yml

```diff
@@ -16,21 +16,24 @@ jobs:
       matrix:
         include:
           - os: macos15
+            conan_profile: default_apple_clang
             build_type: Release
             static: false
           - os: heavy
+            conan_profile: gcc
             build_type: Release
             static: true
             container: '{ "image": "rippleci/clio_ci:latest" }'
           - os: heavy
+            conan_profile: gcc
             build_type: Debug
             static: true
             container: '{ "image": "rippleci/clio_ci:latest" }'
     uses: ./.github/workflows/build_impl.yml
     with:
       runs_on: ${{ matrix.os }}
       container: ${{ matrix.container }}
-      conan_profile: gcc
```

Comment:
This is the reason why we had to patch the name and this value doesn't make sense on Mac and it's easy to make a mistake and use wrong variable